packages/tailwindcss-react-aria-components/src/index.js

```diff
@@ -0,0 +1,50 @@
+import plugin from 'tailwindcss/plugin';
+
+const attributes = {
+  boolean: [
+    'hovered',
+    'focused',
+    'focus-visible',
```

Comment:
Not sure what to do about this, but I think this one might conflict with the native `:focus-visible` selector that tailwind already supports https://tailwindcss.com/docs/hover-focus-and-other-states#focus-visible. Ideally we'd use `data-focus-visible` for RAC but still use native `:focus-visible` for other components, but I don't think Tailwind can differentiate.

Some others here might also conflict, like `disabled`, `empty`, `required`, and `invalid`. Any ideas?

Reply:
would `addVariant('focus-visible', ['&:focus-visible', '&[data-focus-visible]'])` work?

Reply:
That might work, but I'm worried about unintentionally styling an element if `:focus-visible` behavior differs from `data-focus-visible` behavior. If our concern is still supporting both independently, we could introduce something like `native-focus-visible`/`rac-focus-visible` as an escape hatch, but it still seems strange to target both with one variant if they may behave differently for a given element.

Reply:
I've added an option for people to specify a prefix for the variants this plugin creates, in case people would still like to use the native selectors.

Reply:
should we do something like this so that even by default (without a prefix) the native variants still work on non-RAC elements? `[data-rac][data-focus-visible], :not([data-rac]):focus-visible`. We'd need to add a `data-rac` attribute to all of our components so we can identify them. Seems like it would be unexpected for us to completely override the native variants on non-RAC components. I guess we could do that only if no prefix is defined? Adding a prefix would basically be a way to opt out of that and allow using native states on RAC elements.

Reply:
Just updated it to support this, although I can't figure out why the `:not` selector isn't showing up in the output 🤔.

Reply:
Fixed to use `:is` to combine. Also added updated the example app to use this plugin, and added a native input to test that Tailwind's default selector's still work on non-RAC elements.